cli/command/stack/deploy_composefile_test.go

```diff
@@ -56,7 +58,8 @@ func TestValidateExternalNetworks(t *testing.T) {
 			expectedMsg:  "Unexpected",
 		},
 		{
-			network: "host",
+			noInspect: true,
+			network:   "host",
```

Comment:
I'm not rally a fan of this storing state in the testcase struct.

I think the same thing can be accomplished by setting `inspectError`, and asserting there was no error, right?